sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detections/board_detection.py

```diff
@@ -181,6 +183,7 @@ def get_pose(self, model: Optional[CameraModel] = None) -> Tuple[np.array, np.ar
 
     def get_flattened_3d_points(self, model: Optional[CameraModel] = None) -> np.array:
         """Get the image points reprojected into camera coordinates in the 3d space as a (M, 3) array."""
+        # cSpell:ignore reprojected
```

Comment:
We can add `reprojected` to the dictionary.

Reply:
Added it to the respective PR